mkdocs.yml

```diff
@@ -40,4 +39,4 @@ markdown_extensions:
   - toc:
       permalink: true
 
-copyright: © 2018 HL7 & Boston Children's Hospital.<p /> <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
+copyright: © 2018+ HL7 & Boston Children's Hospital. <a rel="license" href="https://cds-hooks.org/license/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
```

Comment:
In the deployed demo this is broken because it's an absolute link. Do you want that instead of...

```suggestion
copyright: © 2018+ HL7 & Boston Children's Hospital. <a rel="license" href="/license/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
```